doc/release/1.5.0-notes.rst

```diff
@@ -80,21 +289,170 @@ Backwards incompatible changes
 
 `scipy.linalg` changes
 ----------------------
+The output signatures of ``?syevr``, ``?heevr`` have been changed from 
+``w, v, info`` to ``w, v, m, isuppz, info``
+
+The order of output arguments ``w``, ``v`` of ``<sy/he>{gv, gvd, gvx}`` is 
+swapped.
 
 `scipy.signal` changes
 ----------------------
+The output length of `scipy.signal.upfirdn` has been corrected, resulting 
+outputs may now be shorter for some combinations of up/down ratios and input 
+signal and filter lengths.
 
 `scipy.stats` changes
 ---------------------
 
 
 Other changes
 =============
+Improved support for leveraging 64-bit integer size from linear algebra backends
+in several parts of the SciPy codebase.
+
+Shims designed to ensure the compatibility of SciPy with Python 2.7 have now 
+been removed.
+
+Many warnings due to unused imports and unused assignments have been addressed.
+
+Early stage adoption of ``mypy`` type hinting in a few parts of the codebase
```

Comment:
It's not specific to mypy. Suggest to change "``mypy`` type hinting" to "type annotations".